sw/SoC/examples/blinky/Makefile

```diff
@@ -1,81 +1,84 @@
-###############################################################################
-# Propject Info
+# Author: Stefano Mercogliano 	<stefano.mercogliano@unina.it>
+# Description:
+#   Makefile for generating a bare-metal software PROGRAM_NAME.
+#	Currently, only rv32 is supported.
 
-PROJECT = filename
-MCU = DEMOSYSTEM
+################
+# Program Name #
+################
 
-###############################################################################
+PROGRAM_NAME = $(shell basename "$$PWD")
```

Comment:
this is brittle, as `pwd` strictly depends on the path from where you launch make. 

E.g. launching `make -C <this dir>`  would breake this whole Makefile

Reply:
You are right. I made a change, give it a double check